smart-metastore/src/main/java/org/smartdata/metastore/utils/MetaStoreUtils.java

```diff
@@ -353,6 +353,23 @@ public static MetaStore getDBAdapter(
           LOG.warn("Database URL not specified, using " + purl);
         }
 
+        if (purl.startsWith(MetaStoreUtils.MYSQL_URL_PREFIX)) {
+          String dbName;
+          String[] dbNameNotAllowed = new String[]{
```

Comment:
This string array can be a final static variable.

Reply:
Yes, I have modified. Thanks!